src/FlightDisplay/FlyViewMap.qml

```diff
@@ -485,8 +485,13 @@ FlightMap {
     QGCMapCircleVisuals {
         id:             orbitTelemetryCircle
         mapControl:     parent
-        mapCircle:      _activeVehicle ? _activeVehicle.orbitMapCircle : null
+        mapCircle:      _activeVehicle ? _activeVehicle.orbitMapCircle : _placeholderMapCircle
         visible:        _activeVehicle ? _activeVehicle.orbitActive : false
+
+        // to prevent null references in QGCMapCircleVisuals
+        QGCMapCircle {
+            id:                 _placeholderMapCircle
+        }
```

Comment:
This should not be needed. QGCMapCircleVisuals code should guard again referring past a null mapCircle.